lib/Models/prettyCoordinates.js

```diff
@@ -0,0 +1,43 @@
+import defined from 'terriajs-cesium/Source/Core/defined';
+import proj4 from 'proj4/lib/index.js';
+
+/**
+ * Turns the latitude / longitude in degrees into a human readable pretty strings.
+ */
+export function prettifyCoordinates(latitude, longitude, height, errorBar) {
```

Comment:
Make the function name match the filename (probably by renaming the file in this case).

Reply:
Done